Directory-Structure.md

```diff
@@ -1,6 +1,13 @@
 Not all files/directories might be available, depending on your setup
 
-## Linux
+The easiest way to see the root folder is to click `Folders` > `View Instance Folder`. Then, go up one level.
+
+MultiMC stores its files in the same location as the MultiMC application.
```

Comment:
leave out the its. Makes the sentence shorter and no worries about a missing ' in a word.